### PR TITLE
Tr 133 remove legacy configuration issue button

### PR DESCRIPTION
**Pull request by DavidNic11**

### CODE REVIEW

File: public/react/Configure/components/Teams/ConfigureTeamsForm.tsx [hunk 1]
```diff
@@ -13,7 +13,6 @@ import InheritanceTextField from "./components/InheritanceTextField";
 import InheritanceToggleField from "./components/InheritanceToggleField";
 import InheritanceSelect from "./components/InheritanceSelect";
 import { RequiredAsterisk } from "./components/Label";
-import { HelperMessage } from "@atlaskit/form";
```
Comment (DavidNic11) [inline public/react/Configure/components/Teams/ConfigureTeamsForm.tsx:16]: unused import

File: public/react/Configure/components/Teams/ConfigureTeams.tsx [hunk 2]
```diff
@@ -89,9 +89,8 @@ const ConfigureTeams: FC<ConfigureTeamsProps> = ({ teamName, jiraFields, onUpdat
         const isLast = index === hierarchyLevels.length - 1;
 
         return (
-          <>
```
Comment (DavidNic11) [inline public/react/Configure/components/Teams/ConfigureTeams.tsx:92]: moved up the key

File: public/shared/velocities-from-issue.js [hunk 3]
```diff
@@ -3,116 +3,117 @@ import { StacheElement, type, ObservableObject } from "../can.js";
...
-                //    teamConfiguration = searchDocument(description, matchTeamTable);
-                
-                return new TeamConfiguration({issue: {...first, url: serverInfo.baseUrl+"/browse/"+first.key}})
-            } else {
-                return new TeamConfiguration({issue: null})
-            }
-
-        })
-        
+  static getTeamConfiguration(jiraHelpers) {
```
Comment (DavidNic11) [inline public/shared/velocities-from-issue.js:7]: This file is all formatting changes, feel free to ignore

**Review by bmomberger-bitovi — APPROVED** [review]
🆗

**Pull request merged**